data/eos-updater-flatpak-installer.service.in

```diff
@@ -13,6 +13,29 @@ Type=oneshot
 RemainAfterExit=true
 ExecStart=@libexecdir@/eos-updater-flatpak-installer
 
+# Sandboxing
+# FIXME: Enable more of these options once we have systemd > 229
+Environment=GIO_USE_VFS=local
```

Comment:
What about `CapabilityBoundingSet` and `RestrictNamespaces`? From looking at `man systemd.exec`, I can’t see any other keys we might want, but please do check yourself!